content/en/hosting/analytics/building-dbt-models.md

```diff
@@ -377,6 +377,13 @@ If Postgres is running in a Kubernetes cluster, you can use the following comman
 kubectl exec -it postgres-pod-name -- psql -U postgres -c "SELECT pg_size_pretty(pg_database_size('your_database_name'));"
 ```
 
+To get the percentage of documents that have synced you can run the following query in your Postgres database:
+```sql
+SELECT (COUNT(*) * 100 / (COUNT(*) + (SELECT SUM(pending) FROM v1.couchdb_progress))) AS sync_percentage FROM v1.medic;
+```
+
+This query selects the total number of documents that have been synced to the `v1.medic` table and divides it by the total number of documents that have been synced and the number of documents that are pending to be synced. This will give you the percentage of documents that have been synced. Update the table names to match the names of the tables in your database making sure to update the schema name as well. Run this query periodically to monitor the progress of the sync and stop the sync process once you get to the desired percentage. It's okay if it is not exactly 10% as long as it is close enough to give you an idea of the disk space required.
```

Comment:
> The default for the POSTGRES_TABLE env variable is couchdb. 

I have updated the query.

> Can we use the env variables keys, rather than the values, so the user is flexible with the names they choose?

The environment variables are set in the debt container and this query is meant to be run directly on the database through a DB client or PSQL on the command line so it would not be possible to use env variables.

Reply:
Can we specify something like: _Please note that the schema and table name could differ according to the environment variables you set_. (and link to the env variables page)?

Reply:
Pushed a new commit with this update.